Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
```

Comment:
### [database_cleaner](http://github.com/DatabaseCleaner/database_cleaner)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.6.1 → 1.7.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **38** commits. See the full changes on [the compare page](https://github.com/DatabaseCleaner/database_cleaner/compare/v1.6.1...v1.7.0).

These are the first 10 commits:
- (db82bee) [Add more advanced exclusion_condition method and reference i…](https://github.com/DatabaseCleaner/database_cleaner/commit/db82bee534f69efba4b5ecb87788ddc593bea128)
- (fdaa552) [Support different column names for MySQL and Postgres](https://github.com/DatabaseCleaner/database_cleaner/commit/fdaa552f6edfbc7f7a68f6440a4d0fcc97c9f8fd)
- (24d592a) [Bumped version of database_cleaner.](https://github.com/DatabaseCleaner/database_cleaner/commit/24d592a674b0e8c49610608635ea9aea895bcb31)
- (c8bc42b) [Remove Gemfile.lock](https://github.com/DatabaseCleaner/database_cleaner/commit/c8bc42b9f669667da1a629026cbb50bd8814a907)
- (ac6964e) [Revert "Remove Gemfile.lock"](https://github.com/DatabaseCleaner/database_cleaner/commit/ac6964eeac245f0ce9e9523e00d16d05f798c399)
- (9755bc5) [Merge pull request #487 from DatabaseCleaner/active-record-e…](https://github.com/DatabaseCleaner/database_cleaner/commit/9755bc55dcb2b2f17df15e4d8fe94772487c2404)
- (b56783f) [Use the right ActiveRecord::Base module.](https://github.com/DatabaseCleaner/database_cleaner/commit/b56783f26741bddfe7c812ec4f014def32edfc2b)
- (40e084e) [Merge pull request #490 from mauro-oto/use-right-activerecor…](https://github.com/DatabaseCleaner/database_cleaner/commit/40e084e9a903fdae186ad4196bb5b306498a5096)
- (0cc3991) [Fixed typo](https://github.com/DatabaseCleaner/database_cleaner/commit/0cc3991ea961fd42064e9baaa453468ce7ddb304)
- (d746896) [Update README to indicate support for Sequel deletion strate…](https://github.com/DatabaseCleaner/database_cleaner/commit/d7468962a791dcef1b95d7f7def751194ba120be)


</details>